src/Streamly/Coreutils/Mkdir.hs

```diff
@@ -0,0 +1,21 @@
+-- |
+-- Module      : Streamly.Coreutils.Mkdir
+-- Copyright   : (c) 2022 Composewell Technologies
+-- License     : Apache-2.0
+-- Maintainer  : streamly@composewell.com
+-- Stability   : experimental
+-- Portability : GHC
+--
+-- Create the Directory(ies), if they do not already exist.
+
+module Streamly.Coreutils.Mkdir where
```

Comment:
Use the following warning options in the cabal file:

```
    ghc-options:        -Weverything
                        -Wno-implicit-prelude
                        -Wno-prepositive-qualified-module
                        -Wno-missing-import-lists
                        -Wno-missing-safe-haskell-mode
                        -Wno-unsafe
                        -Wno-missing-deriving-strategies
                        -Wno-monomorphism-restriction
                        -Wno-missing-local-signatures
                        -Wno-missed-specialisations
                        -Wno-all-missed-specialisations 
```

Reply:
fixed in PR https://github.com/composewell/streamly-coreutils/pull/60